<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GrabCutWindow</class>
 <widget class="QMainWindow" name="GrabCutWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>22</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GrabCut</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>431</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>loadImage</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>160</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="x1LineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="y1LineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="x2lineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="y2LineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>330</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>run GrabCut1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Please enter the grabcut position</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadbutton2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>410</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>loadImage2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runbutton2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>400</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>run GrabCut2</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
